docs/user/settings.md

```diff
@@ -80,3 +80,16 @@ dashboard: {
 }
 ```
 
+#### `maxHttpBufferSize`
+
+This set the maximum message size (in bytes) the socket can send.
+Default value is 1MiB (1E6 bytes).
+
+Change this value to allow larger files to be uploaded.
+
+```js
+dashboard: {
+    maxHttpBufferSize: 1e8 // size in bytes, example: 100 MB
+}
+```
```

Comment:
Where is this setting to be made Joe?

Node-RED settings.js file? Top level?

Not being pedantic, i honestly dont know where 🤔 (will probably come to me immediately after posting this)

Reply:
Just realised this was not @joepavitt PR. Oops.

Reply:
No problem @Steve-Mcl.
This setting is for Node-RED settings.js, the whole file you cite documents configuration appended in Node-RED settings.js. 
I tried to follow the way already used (for custom middlewares and other extensions).

Reply:
@Ek1nox has rightly extended our existing configuration defined in https://dashboard.flowfuse.com/user/settings.html as an fyi @Steve-Mcl